tests/e2e/specs/modules/adsense/setup-new-user.test.js

```diff
@@ -196,6 +200,8 @@ describe( 'setting up the AdSense module', () => {
 
 		await proceedToAdsenseSetup();
 
+		await page.waitForSelector( '.googlesitekit-setup-module--adsense' );
```

Comment:
We could move this to `proceedToAdsenseSetup` if it's needed for every test?

Reply:
Actually, it was already in there… so I'm gonna remove it. Might've slipped in during debug, cheers!